tasks/ca/statcan/data.py

```diff
@@ -53,6 +53,22 @@ def download(self):
 
 
 class SplitAndTransposeData(BaseParams, Task):
+    INDIAN_RESERVES_SUFFIX = '-DQ'
```

Comment:
Here you can find the data quality information: http://www12.statcan.gc.ca/census-recensement/2016/dp-pd/prof/details/Page.cfm?Lang=E&Geo1=PR&Code1=62&Geo2=&Code2=&Data=Count&SearchText=Nunavut&SearchType=Begins&SearchPR=01&B1=All&GeoLevel=PR&GeoCode=62#dqf1-popup

Maybe it would be better to change INDIAN_RESERVES_SUFFIX to DATA_QUALITY_SUFFIX